### ISSUE TITLE
Remove some of the warnings of process-docs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2530]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#2530|
|Type|defect|
|Created|2007-03-20 10:01:52Z|

process-docs complains about some undefined references during pdf generation. Attached patch corrects the problem.

Attachments:

* [quotes_2530.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b94/b94057166f8748d644a37d641e547a003f4f66ca/437aa12e83a73f60edb9523ee9b610a4c61ffd4b.diff) (1387 bytes) - added by therve on 2007-03-20 10:02:27Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__2530 2530
type__defect defect
reporter__therve therve
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1174384912000000 1174384912000000
changetime__1174391120000000 1174391120000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(Note this is a duplicate of [#11044](https://github.com/twisted/twisted/issues/11044), since there's no patch there I'm closing that one as a duplicate)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @therve|
|-|-|

Thanks for figuring this out.  Change looks good, fixes the latex errors.  It's probably not quite ideal that the files appear inline in the chapter (rather than in an appendix), but that's a matter for another day...

The only other issue here is that the TwistedQuotes example is woefully out of date :P  Someone should update it to use the new plugin system.  That doesn't mean this shouldn't be applied though, so please apply it. :)

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set status to `closed`|
|-|-|

(In [19880]) Fix latex errors in TwistedQuotes documentation.

Author: therve
Reviewer: exarkun
Fixes [#2530](https://github.com/twisted/twisted/issues/2530)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
